tabs_extra.py

```diff
@@ -44,6 +47,15 @@
 ###############################
 # Helpers
 ###############################
+def path2url(path):
```

Comment:
I don't think we need a separate function for this one line operation since it won't be used anywhere else.

Also, I don't think we need to carry around the stackoverflow comment as this is a straight forward use of the library.